companion/src/simulation/simulatordialog.cpp

```diff
@@ -220,6 +222,13 @@ void SimulatorDialog::keyPressEvent (QKeyEvent *event)
     case Qt::Key_PageUp:    
       buttonPressed = event->key();
       break;
+    case Qt::Key_Z:
```

Comment:
On FR keyboards this is not good, perhaps is it possible you check the keyboard layout?